test/integration/serverless/snapshots/log-java

```diff
@@ -194,6 +194,51 @@
     "ddsource": "lambda",
     "ddtags": "account_id:425362996713,architecture:XXX,aws_account:425362996713,dd_extension_version:123,env:integration-tests-env,function_arn:arn:aws:lambda:eu-west-1:425362996713:function:integration-tests-extension-XXXXXX-log-java,functionname:integration-tests-extension-XXXXXX-log-java,memorysize:1024,region:eu-west-1,resource:integration-tests-extension-XXXXXX-log-java,runtime:java8.al2,service:integration-tests-service,taga:valuea,tagb:valueb,tagc:valuec,tagd:valued,version:integration-tests-version"
   },
+  {
+    "message": {
+      "message": "[dd.trace TIMESTAMP +0000] [main] ERROR datadog.trace.lambda.LambdaHandler - could not notify the extension as the lambda span is null or no sampling priority has been found\n",
```

Comment:
This is an indication of a real issue with the newest version of the tracing layer. It will be addressed in a separate PR.